src/app/modules/item/components/item-chapter-view/group-situation-chapter-view/group-situation-chapter-view.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { forkJoin, merge, Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Group } from 'src/app/core/components/group-nav-tree/group';
+import { fetchingState, isReady, readyState } from 'src/app/shared/helpers/state';
+import { GetGroupUserDescendantsService } from 'src/app/shared/http-services/get-group-user-descendants.service';
+import { GetGroupUsersProgressService, GroupUsersProgress } from 'src/app/shared/http-services/get-group-users-progress.service';
+import { GetItemChildrenService, ItemChild } from '../../../http-services/get-item-children.service';
+import { ItemData } from '../../../services/item-datasource.service';
+
+interface User {
```

Comment:
I think it would be better to make `UserInfos` from UserPipe public (that I would call "FormattableUser") and use it here instead of redefining the same.

Reply:
fixed